pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -301,6 +303,12 @@ func generateCloudConfig(configFile *ConfigFile) (cloudConfig *CloudConfig, err
 // If no tokenSource is specified, uses oauth2.DefaultTokenSource.
 // If managedZones is nil / empty all zones in the region will be managed.
 func CreateGCECloud(config *CloudConfig) (*GCECloud, error) {
+	// Remove any pre-release version and build metadata from the semver, leaving only the MAJOR.MINOR.PATCH portion.
+        // See http://semver.org/.
+	version := strings.Split(strings.Split(version.Get().GitVersion, "-")[0], "+")[0]
+
+        // Create a user-agent header append string to supply to the Google API clients, to identify Kubernetes as the origin of the GCP API calls.
+	userAgent := fmt.Sprintf("(%s %s) Kubernetes/%s", runtime.GOOS, runtime.GOARCH, version)
```

Comment:
Could we re-order the args to "Kubernetes/%s (%s %s)"?

Reply:
Let me get back to you on this.

Reply:
Any update on this? Putting '<product>/<product-version>' first seems to be the standard form.

Reply:
I re-arranged the order of the components. I'm not quite sure what you mean about putting '/' first. There's a '/' between the component name and component version, which I believe is standard. The component details are included in parentheses.

Reply:
I mean component and component version come before platform details. 
For example `Googlebot/2.1 (+http://www.google.com/bot.html)`